plugins/ratelimit.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const rateLimit = require('hapi-rate-limit');
+const config = require('config');
+const rateLimitConfig = config.get('rateLimit');
+
+module.exports = {
+    name: 'ratelimit',
+    async register(server) {
+        server.ext('onPreAuth', async (request, h) => {
+            // see https://github.com/glennjones/hapi-swagger/issues/623
+            if (request.path.startsWith('/v4/swagger')) {
```

Comment:
should we store all whitelisted paths in an array config, and handle in preAuth hook, rather than adding to individual plugins like in the rest of the files in this PR ?

Reply:
Keeping the configuration close to where the route is defined promote integrity similar to how `auth` is defined for each route. Also, one less place to change if certain routes change to require authenticate. This swagger hook you are seeing is due to current limitation on hapi-swagger.